nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -253,6 +253,8 @@
 
 - `zig_0_9` and `zig_0_10` have been removed, you should upgrade to `zig_0_13` (also available as just `zig`), `zig_0_12` or `zig_0_11` instead.
 
+- `webpack-cli` was updated to version 6.0.1, which has breaking changes from the previous version 5.1.4. See the [upstream release notes](https://github.com/webpack/webpack-cli/releases/tag/webpack-cli%406.0.0) for details on these changes.
```

Comment:
```suggestion
- `webpack-cli` was updated to major release 6, which has breaking changes from the previous version 5.1.4. See the [upstream release notes](https://github.com/webpack/webpack-cli/releases/tag/webpack-cli%406.0.0) for details on these changes.
```
in case we have another update